test.py

```diff
@@ -0,0 +1,15 @@
+import lilcomlib
+import numpy
+
+# Sample test array to test the function.
+numpy_array = numpy.array([[1]* 3]*100)
+
+# Running the function in lilcommodule.c
+val1 = lilcomlib.compress(numpy_array)
+
+# numpy_array = numpy.array([[5]*10] * 100)
+# val2 = lilcomlib.decompress(numpy_array)
```

Comment:
I assume this means you haven't got the decompression working yet?

Reply:
No the decompression function is working. I just forgot to commit the modifications in `test.py`. Now I committed and it has been changed